packages/datadog-plugin-pg/src/index.js

```diff
@@ -71,6 +78,14 @@ function createWrapQuery (tracer, config) {
   }
 }
 
+function normalizeConfig (config) {
+  return Object.assign({}, config, {
+    includeQueryParams: typeof config.includeQueryParams === 'function'
+      ? config.includeQueryParams
+      : config.includeQueryParams === true ? p => p : undefined
```

Comment:
What happens if `includeQueryParams` is false and this returns `undefined` to the caller instead of a function?